app/components/SidebarMenu/SidebarMenu.component.js

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { Menu, Sidebar, Icon } from "semantic-ui-react";
+import { NavLink } from "react-router-dom";
+import ESTRChallengeSidebar from "./images/estr-sidebar-logo.png";
+import "./style.scss";
+
+const SidebarMenu = (props) => {
+  return (
+    <Sidebar.Pushable className="sidebar">
+      <Sidebar
```

Comment:
If you add `onClick={() => setSidebarVisible(false)} `to `SideBar` then you won't need it in the menu items.